include/streaming_protocol/SubscribedSignal.hpp

```diff
@@ -231,6 +231,9 @@ class SubscribedSignal {
     std::string m_timeBaseEpochAsString;
     uint64_t m_timeBaseFrequency;
     Unit m_unit;
+    Range m_range;
```

Comment:
It would be nice to have getter functions for the range and postScaling member values

Reply:
I will add those

Reply:
Added them!